daemon/containerd/image_exporter.go

```diff
@@ -266,16 +286,19 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, platf
 		}),
 	}
 
-	if platform == nil {
+	if !specificPlatforms {
 		// Allow variants to be missing if no specific platform is requested.
 		opts = append(opts, containerd.WithSkipMissing())
 	}
 
 	imgs, err := i.client.Import(ctx, decompressed, opts...)
 	if err != nil {
-		if platform != nil {
-			p := platforms.FormatAll(*platform)
-			log.G(ctx).WithFields(log.Fields{"error": err, "platform": p}).Debug("failed to import image to containerd")
+		if specificPlatforms {
+			platformNames := make([]string, 0, len(platformList))
+			for _, p := range platformList {
+				platformNames = append(platformNames, platforms.FormatAll(p))
+			}
+			log.G(ctx).WithFields(log.Fields{"error": err, "platform(s)": platformNames}).Debug("failed to import image to containerd")
```

Comment:
Nit (can be updated in a follow-up); `platform(s)` as name for the structured logs feels a bit awkward; probably either `platforms` or `platform` would be better.